docs/index.md

```diff
@@ -0,0 +1,64 @@
+# Index
+
+-   [Blogs](https://github.com/spiffe/tornjak/blob/main/docs/blogs.md)
```

Comment:
in general for all the links, you can do relative links, which is better so that links don't always point to the main branch. I believe this is how it works, but definitely test the links. 

```suggestion
-   [Blogs](/docs/blogs.md)
```